pycht/pycht.py

```diff
@@ -18,7 +18,7 @@ def __init__(self) -> None:
         self.image_processing = ImageProcessing()
         self.clustering = Clustering()
 
-    def stencil(self, input_img: str, nb_colors: int=4, output_path: str="./") -> None:
+    def stencil(self, input_img: str, nb_colors: int = 4, output_path: str = "./") -> None:
```

Comment:
I think you wanted to update the default number of clusters to **3**.

```suggestion
    def stencil(self, input_img: str, nb_colors: int = 3, output_path: str = "./") -> None:
```